Vizzuality/heco-invest
# HOTFIX - Hide the 2FA user preference

**⚠️ DO NOT MERGE THIS PR. ⚠️**

This PR contains a hotfix that hides the 2FA setting from the user preferences so that account users are not locked out of their  accounts because the feature is incomplete in v1.4.0.

Only the last 2 commits of this branch are relevant. The others are the commits that will be merged into `main` tomorrow morning when we deploy v1.4.0 to production.

The last commit, “Release v1.4.1”, is tagged as v1.4.1. After this PR is merged, I will create a new release and also merge manually the changes to `develop`.

## Testing instructions

Only review the last 2 commits. Make sure that the account users can't enable/disable 2FA.

## Tracking

[LET-1225](https://vizzuality.atlassian.net/browse/LET-1225)
